composer.json

```diff
@@ -19,6 +19,7 @@
         "php": ">= 5.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.0.*"
+        "phpunit/phpunit": "4.0.*",
+        "squizlabs/php_codesniffer": "^3.1"
```

Comment:
not sure if you want to add this dependency here.

It makes the `phpcs` and `phpcbf` commands available for things like:
`php vendor/bin/phpcbf ./ --standard=PSR2 --ignore=vendor` to auto-fix some things
or `php vendor/bin/phpcs ./ --standard=PSR2 --ignore=vendor` to show remaining errors

Reply:
I think it's a good idea to have this if we are to enforce PSR2 in future pull requests. 👍